django_tenants_celery_beat/models.py

```diff
@@ -1,22 +1,25 @@
 import json
 
-from django.db import models
-from django_celery_beat.models import PeriodicTask, CrontabSchedule
 import pytz
 import timezone_field
-
 from django.conf import settings
-from django_tenants.utils import get_tenant_model, get_public_schema_name
-from django_tenants_celery_beat.utils import get_periodic_task_tenant_link_model
+from django.db import models
+from django_celery_beat.models import CrontabSchedule, PeriodicTask
+from django_tenants.utils import get_public_schema_name, get_tenant_model
+
+from django_tenants_celery_beat.utils import \
+    get_periodic_task_tenant_link_model
+
+timezone_field_kwargs = {
+    "default": "UTC",
+}
+
+if getattr(settings, "TENANT_TIMEZONE_DISPLAY_GMT_OFFSET", False):
+    timezone_field_kwargs["choices_display"] = "WITH_GMT_OFFSET"
 
 
 class TenantTimezoneMixin(models.Model):
-    timezone = timezone_field.TimeZoneField(
-        default="UTC",
-        display_GMT_offset=getattr(
```

Comment:
Thank you for submitting this PR.

If you can clean up this commit to:
- remove formatting changes
- reword the commit so the summary line starts `fix: ...` (e.g. `fix: replace deprecated TimeZoneField kwarg`)

then the PR will be accepted.